snippets/mapped-table.md

```diff
@@ -0,0 +1,45 @@
+### MappedTable
+
+Renders a table with rows dynamically created from an array of data.
+
+Use the value of the `headers` prop to conditionally render a table with a header. Use `Array.prototype.map` to render every item in `data` as a `<tr>` element and give it a `key`. `data` can either be an array of objects with the `id` and `value` properties or an array of primitives. Omit the `headers` prop to render a `table` without headers.
+
+```jsx
+function MappedTable(props) {
+  return (
+    <table>
+      <thead>
+        {props.headers ? (
```

Comment:
Maybe `props.header` if there's only one

Reply:
Yep, agree with this one